### DISCUSSION TITLE
Welcome to Quansheng_UV-K5_Wiki Discussions!

### DISCUSSION THREAD

**Discussion by ludwich66** [ludwich66/Quansheng_UV-K5_Wiki#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RE3CON** [comment]
can help to write the [German Manual](https://github.com/RE3CON/K5-Wiki/blob/master/Manual_DE.md) better complied. See diffs.
73s
Chris

**Reply by ludwich66** [reply]
Hallo Chris,
das deutsche Handbuch wurde zuerst aus dem Orginalhandbuch übersetzt. Im Moment wird die englische Version von "Graydesk" überarbeitet, die Beschreibung der Funktionen wird geprüft und ergänzt bzw. besser beschrieben. Es wäre schön, wenn du die deutsche Anleitung mit den überarbeiteten Texten der Englischen Version abgleichen/aufhübschen würdest. 
Ich schau mir die Diffs nachher mal durch.
Ich kann dir auch die Mutter der Webversion anbieten:
[Translate_01_de.odt](https://github.com/ludwich66/Quansheng_UV-K5_Wiki/files/12243007/Translate_01_de.odt)

Jedenfalls vielen Dank für deine Hilfsbereitschaft.

Ludwich

**Comment by DurvalMenezes** [comment]
I'm an opensource enthusiast who recently passed my ham exam, and have already purchased my (of course!) Quansheng UV-K5.

Have good C programming skills plus a couple of other abilities which I do not think will apply (good PERL and BASH scripting plus Linux sysadmin abilities, plus modest Python coding skills). 

Speak good English, excellent Portuguese and modest Spanish, so I can help with translations to these languages.

Please let me know if I can be of aid.

**Reply by Tunas1337** [reply]
Hello! Nice to meet you.
Yes, we would definitely need quite a lot of help in the area of C programming for the [open-source recreation of the K5's firmware](https://github.com/DualTachyon/uv-k5-firmware), and also potentially [Piotr's parallel-executing applications.](https://github.com/piotr022/UV_K5_playground)
Check out those repos and see if anything intrigues you. Thanks again for offering your help!

**Reply by DurvalMenezes** [reply]
Thanks for the warm welcome!

> Check out those repos and see if anything intrigues you.

Both are very intriguing, but I think the opensource recreation would probably have more impact: imagine that, having an inexpensive, multiband, fully opensource radio! I plan on installing a development environment here and trying to build and install it myself, as soon as I get my 3rd radio (which I'm going to dedicate to these projects). 

Thanks for the suggestions!

**Comment by RE3CON** [comment]
Hi
Habs am Tablet mit Android als .dotx nur speichern können. Vielleicht kann
ich trotzdem mithelfen und das ein oder andere ist zum ergänzen geeignet.

73 / Grüsse
Chris
[Translate_01_de.zip](https://github.com/ludwich66/Quansheng_UV-K5_Wiki/files/12334205/Translate_01_de.zip)

**Reply by ludwich66** [reply]
Hallo Chris,
jetzt hab ich grad nochmals hier vorbeigeschaut, ich dachte ich hätte meine Antwort schon verschickt ...
Wo hast du denn deine Anpassung im .dotx Format liegen?
Grüße ludwich

**Comment by RE3CON** [comment]
auf pmr mod NFM 12,5khz statt WFM etc, etc.... Muss am PC das doch in odt statt dotx bezuegl. diff merge am besten machen statt auf mein handy android pad.
73s Gruesse
Chris

**Comment by mareksbl** [comment]
No sound after installing mod 5 full band on  R51 v1.5, any ideas?

**Reply by ludwich66** [reply]
Please use Telegramm Groups to ask
@quansheng_uvk5_en
https://t.me/c/quansheng_uvk5_en_dev/29985
